packages/fast-components-react-base/src/listbox/listbox.tsx

```diff
@@ -211,10 +265,21 @@ class Listbox extends Foundation<
     }
 
     public componentDidMount(): void {
-        const children: Element[] = this.domChildren();
-        const focusIndex: number = children.findIndex(this.isFocusableElement);
+        let focusIndex: number = -1;
 
+        if (this.state.selectedItems.length > 0) {
+            focusIndex = Listbox.getItemIndexById(
+                this.state.selectedItems[0].id,
+                this.props.children
+            );
+        } else {
+            const children: Element[] = this.domChildren();
+            focusIndex = children.findIndex(this.isFocusableElement);
+        }
```

Comment:
nit: we usually use a ternary here and since there is no need to store children you could instead do:

```
const focusIndex: number = this.state.selectedItems.length > 0
    ? this.getItemIndexById(
           this.state.selectedItems[0].id,
           this.props.children
       );
    : this.domChildren().findIndex(this.isFocusableElement);
```

Reply:
Done